llama_index/indices/vector_store/retrievers/retriever.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         self._doc_ids = doc_ids
         self._filters = filters
 
-        self._kwargs: Dict[str, Any] = kwargs.get("vector_store_kwargs", {})
+        self._kwargs: Dict[str, Any] = kwargs
```

Comment:
This seems like a breaking change. `vector_store_kwargs` was how you were able to pass filters and other parameters